### PR TITLE
Ensure consistent Navbar links across apps

### LINKED ISSUE #7: Improve cage porosity model app
The current cage porosity model app is a placeholder to be improved. 

TODO

- [x] Replace the manually written front end HTML and javascript with a (hopefully simple) Dash app.
- [x] Allow for other input types: (i) file upload (.mol) (ii) drawing a structure using [jsml](https://dash.plotly.com/dash-bio/jsme) or similar, if possible.

### PR DESCRIPTION
**Pull request by AdrianDAlessandro**
This PR provides a way to keep the Navbar consistent between all of the apps and link between them. The main detail is that the `app_list.json` file needs to be central too all the containers (I assume this is possible by mounting it as volume to all of them), and also an APP_NAME variable should defined for each container.

This should be merged and then used to make the final cage porosity app before #23 is merged.

### CODE REVIEW

**Comment by AdrianDAlessandro** [comment]
Screenshot 2022-05-27 at 17 02 02

File: website/app.py [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
 
 from . import callbacks  # noqa: F401
 from .layout import content
 
-url_prefix = os.getenv("URL_PREFIX", "")
+URL_PREFIX = os.getenv("URL_PREFIX", "")
+APP_NAME = os.getenv("APP_NAME", "")
+
+if (filepath := Path(__file__).parent.parent / "app_list.yaml").exists():
+    with open(filepath, "r") as f:
```
Comment (dandavies99) [inline website/app.py]:
```suggestion
if (Path(__file__).parent.parent / "app_list.yaml").exists():
    filepath = Path(__file__).parent.parent / "app_list.yaml"
    with open(filepath, "r") as f:
```

Comment (dandavies99) [reply website/app.py]: The walrus operator is ideal here, but not allowed in Python 3.6 😢

**Review by dandavies99 — CHANGES_REQUESTED** [review]
This all looks good to me. I suggest we also add 
```bash
environment:
      - APP_NAME="Is my cage porous?"
```
to `docker-compose.yml` as an example of how the APP_NAME variable would be declared.

**Pull request merged**